test/setup_mongo_wendy.js

```diff
@@ -0,0 +1,14 @@
+var DB = require('mongodb').Db;
+var Server = require('mongodb').Server;
+var db = new DB('test', new Server(process.env.MONGODB_HOST || "localhost", 27017));
+db.open()
+  .then(d => d.admin())
+  .then(a => a.addUser('admin','admin'))
+  .then(x => {
```

Comment:
I think this could just be `.then(console.log)`, since it should exit with 0 anyway. You don't need to change it if you don't want to, just pointing it out.